tests/online/test_02_import_export.py

```diff
@@ -63,25 +63,22 @@ def test_import_export(helpers: Helpers):
 
     # Import a single issue.
     ret = helpers.run_pyjiracli(
-        credentials + ["import", "./examples/import_issues/single_issue.json"])
-
+        ["import"] + credentials + ["./examples/import_issues/single_issue.json"])

```

Comment:
Why aren't the credentials just inside the argument list?

Reply:
'credentials' is a constant (kind of) which is used multiple times in this function. Not sure I understood your question.

Reply:
Is the following possible?
```suggestion
        ["import", credentials , "./examples/import_issues/single_issue.json"])

```

Reply:
No. List cannot be used in the initializer here.